smurfs/src/views/smurfslistview.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { connect } from "react-redux";
+
+
+import {getSmurfs} from '../actions/index'
+import SmurfsList from "../components/smurfslist";
+import { SmurfForm } from "../components";
+
+
+class SmurfsListView extends React.Component {
+  constructor() {
+    super();
+  }
+
+  componentDidMount() {
+   
+    this.props.getSmurfs();
+    
+  }
+
+  render() {
+    if (this.props.fetching) {
+  
```

Comment:
This is where you would put a loader component or `getting smurfs...` or something like that